Lagerregal/settings/development.py

```diff
@@ -53,9 +53,9 @@
 # EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
 try:
-    INSTALLED_APPS += ['debug_toolbar']
-    MIDDLEWARE = [
+    INSTALLED_APPS += ('debug_toolbar', )
+    MIDDLEWARE = (
         'debug_toolbar.middleware.DebugToolbarMiddleware',
-    ] + MIDDLEWARE
+     ) + MIDDLEWARE
```

Comment:
We should change the values in `base.py` to lists, not the other way around.

Source: [guido](https://marc.info/?l=python-dev&m=107666578526990&w=2), [django mailing list](https://groups.google.com/d/msg/django-developers/h4FSYWzMJhs/_2iVc4qgfsoJ)